test/unit/test_qvm.py

```diff
@@ -59,7 +60,8 @@ def test_qvm_run_region_declared_not_measured(client_configuration: QCSClient):
     p = Program(Declare("reg", "BIT"), X(0))
     result = qvm.run(p.wrap_in_numshots_loop(100))
     bitstrings = result.readout_data.get("reg")
-    assert bitstrings.shape == (100, 0)
+    assert bitstrings is not None
+    assert bitstrings.shape == (100, 1)
```

Comment:
See [this comment](https://github.com/rigetti/pyquil/pull/1579/files#r1201209257) for why the shape here is different.